feature_engine/transformation/ciclycal.py

```diff
@@ -0,0 +1,122 @@
+from typing import List, Optional, Union
+import numpy as np
+import pandas as pd
+
+
+from feature_engine.base_transformers import BaseNumericalTransformer
+from feature_engine.variable_manipulation import _check_input_parameter_variables
+
+
+class CyclicalTransformer(BaseNumericalTransformer):
+    """
+    The CyclicalTransformer() applies a Ciclycal transformation to numerical
+    variables.
+
+    There are some feature that are cyclic by nature. One example of this are
+    the hours of a day or the months of a year. In both cases the higher values of
+    a set of data are closer to the lower values of that set.
+
+    For example the month October (10) is closer to month January (1) than
+    to month February (2).
+
+    To check an explanation about this:
+    http://blog.davidkaleko.com/feature-engineering-cyclical-features.html
+
+    The CyclicalTransformer() works only with numerical variables. But does not
+    allow null values on any row of original column
+
+    A list of variables can be passed as an argument. Alternatively, the
+    transformer will automatically select and transform all numerical
+    variables.
+
+    Parameters
+    ----------
+    variables : list, default=None
+        The list of numerical variables that will be transformed. If None, the
+        transformer will automatically find and select all numerical variables.
+
+    Methods
+    -------
+    fit:
+        This transformer does not learn parameters.
+    transform:
+        Apply the CyclicalTransformer transformation.
+    fit_transform:
+        Fit to data, then transform it.
+
+    References
+    ----------
+    ..
+    """
+
+    def __init__(
+            self, variables: Union[None, int, str, List[Union[str, int]]] = None
+    ) -> None:
+        self.variables = _check_input_parameter_variables(variables)
+
+    def fit(self, X: pd.DataFrame, y: Optional[pd.Series] = None):
+        """
+        This transformer does not learn any parameter. But keeps a map of the max values
```

Comment:
Does this not learn the maximum value to be able to convert the features into cyclical?

Reply:
Yeah you are right I forgot to change docstrings after i implemented the maximum value.